crates/ruff_formatter/Cargo.toml

```diff
@@ -11,7 +11,7 @@ cfg-if = { version = "1.0.0" }
 countme = { version = "3.0.1" }
 drop_bomb = { version = "0.1.5" }
 indexmap = { version = "1.9.1" }
```

Comment:
Did you verify if all these dependencies are still needed? https://github.com/est31/cargo-udeps

Reply:
Was able to remove a handful of them -- good call.